src/Documents/Session/DocumentSessionRevisions.ts

```diff
@@ -15,8 +15,9 @@ import { RevisionsCollectionObject } from "../../Types";
 import { ForceRevisionStrategy } from "./ForceRevisionStrategy";
 import { throwError } from "../../Exceptions/index";
 import { StringUtil } from "../../Utility/StringUtil";
+import {DocumentSessionRevisionsBase} from "./DocumentSessionRevisionsBase";
 
-export class DocumentSessionRevisions extends AdvancedSessionExtensionBase implements IRevisionsSessionOperations {
+export class DocumentSessionRevisions extends DocumentSessionRevisionsBase implements IRevisionsSessionOperations {
```

Comment:
Do you still need AdvancedSessionExtensionBase in the imports?

Reply:
removed